docs/security-spec.md

```diff
@@ -0,0 +1,1080 @@
+﻿# Security Specification
+
+#### 4. Security
+   - 4. Security
```

Comment:
can we make the TOC link to the respective sections, please? Also, why are we starting a new document with a section numbered 4?